test/util/IdTableHelpers.h

```diff
@@ -106,17 +106,22 @@ IdTable generateIdTable(
 
 /*
 @brief Create an `IdTable`, where the content of the join columns are given via
-repeatedly called generator functions (one function per join column).
+repeatedly called generator functions (one function per join column) and other
+entries are random.
 
 @param numberRows numberColumns The number of rows and columns, the table should
 have.
 @param joinColumnWithGenerator Every pair describes the position of a join
 column and the function, which will be called, to generate it's entries.
+@param randomNumberGeneratorSeed The seed for the random number generator, that
+generates the content for the non join column entries.
 */
 IdTable createRandomlyFilledIdTable(
     const size_t numberRows, const size_t numberColumns,
     const std::vector<std::pair<size_t, std::function<ValueId()>>>&
-        joinColumnWithGenerator);
+        joinColumnWithGenerator,
+    const unsigned int randomNumberGeneratorSeed =
```

Comment:
Also, once the `seed` is a strong type, you can also rename the arguments to `seed` or `randomSeed`

Reply:
Done, see [abbbe85](https://github.com/ad-freiburg/qlever/pull/1126/commits/abbbe85a8dd2130b9832263b6d18f9ae46d3e6a2).